Makefile

```diff
@@ -196,3 +186,7 @@ $(eval $(call linter,clang-format))
 $(eval $(call linter,shellcheck))
 $(eval $(call linter,shfmt))
 $(eval $(call linter,circleci_validate))
+
+.PHONY: linters
+linters:
```

Comment:
could we use `pre-commit` to do this? i.e. `pre-commit run --all-files`

Reply:
We would need to install `pre-commit` in the apollo-ci image, can we do it in a followup?

Reply:
sure - I might find some time this week to do so